addons/l10n_gcc_invoice/views/report_invoice.xml

```diff
@@ -447,6 +447,17 @@
                     </div>
                 </div>
 
+                <div class="mb-2">
+                    <p class="lh-sm" t-if="o.company_id.display_invoice_amount_total_words">
+                        Total amount in words: <br/>
+                        <span class="text-muted" t-field="o.amount_total_words"/>
+                    </p>
+                    <p class="text-end lh-sm" t-if="o.company_id.display_invoice_amount_total_words">
+                        المبلغ الإجمالي <br/>
+                        <span class="text-muted" t-field="o.with_context(show_currency_unit_label=False).amount_total_words_arabic"/>
```

Comment:
hi, wouldn't this work? without adding a field? :slightly_smiling_face: 
```suggestion
                        <span class="text-muted" t-field="o.with_context(show_currency_unit_label=False, lang='ar_001').amount_total_words"/>
```

Reply:
It wouldn't work as is, because the field would be computed only once, but we have to display the field twice with different languages. However, we could make it work with a`@api.depends_context('lang')`.